DataSetExplorer/Controllers/Dataset/DataSetController.cs

```diff
@@ -0,0 +1,52 @@
+﻿using DataSetExplorer.Controllers.Dataset.DTOs;
+using DataSetExplorer.DataSetBuilder.Model;
+using FluentResults;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DataSetExplorer.Controllers.Dataset
+{
+    [Route("api/dataset/")]
+    [ApiController]
+    public class DataSetController : ControllerBase
+    {
+        private readonly string _cloneGitPath = "../../ClonedProjects/";
```

Comment:
Should be read from configuration, e.g., https://medium.com/@dozieogbo/a-better-way-to-inject-appsettings-in-asp-net-core-96be36ffa22b